docs/building-a-form/available-components.md

```diff
@@ -348,18 +416,27 @@ When you build a form with more than one chapter (shown by the segments in a pro
 
 #### Usage guidelines
 
-Break into separate topic?
+In general, the review page renders the form data in review mode automatically. There are, however, some specific options you can pass to the form config to customize certain review functionality.
+
+This property is nested directly under `uiSchema`:
+- `'ui:reviewWidget'`: takes a widget component to render on the review page for that field. Default review widgets are automatically rendered, so only use this if you need to customize the review widget that is used.
+
+These properties are nested under `uiSchema: { `ui:options`: {} }`:
+- `hideOnReview`: Hides the field on the review page; takes a `boolean`
+- `hideOnReviewIfFalse`: Hides the specified field on the review page when the field value is `false`; takes a `boolean`
+- `keepInPageOnReview`: By default, array fields that are displayed on a single page in a form, such as information for multiple dependents, are displayed in a separate section on the review page. To keep the information in a single section on a review page, set this property.
+
+Used when you have an array field that should not be pulled out of the page its in and shown separately on the review page; takes a `boolean`
```

Comment:
This paragraph doesn't appear tied to any other information?

Reply:
oops! meant to delete that, thanks for catching!